frappe/core/doctype/prepared_report/test_prepared_report.py

```diff
@@ -63,9 +63,12 @@ def test_prepared_data(self):
 		self.assertEqual(len(prepared_data["result"]), len(generated_data["result"]))
 		self.assertEqual(len(prepared_data), len(generated_data))
 
-	@run_only_if(db_type_is.MARIADB)
 	def test_start_status_and_kill_jobs(self):
-		with test_report(report_type="Query Report", query="select sleep(10)") as report:
+		if db_type_is.POSTGRES:
```

Comment:
This is just an enum, not supposed to be a boolean expression, it'll always be true.

Should check `frappe.conf.db_type` (or `frappe.db.db_type`)